### ISSUE TITLE
Not Found 404

### ISSUE THREAD

**Issue by zetneteork** [Radarr/Radarr#3796]
**Bug:**
Previously added movies are not able to access. It is also not possible to remove movie.
This happens not for all movies, just couple of them. It is not issue with permissions. It is not issue with cookies, tested on different browsers or direct API via mobile app `nzb360`

**To reproduce**
1. Add new movie
2. Go to the new movie, unable to access
3. Unable to use search for movie, it is only visible when the movies is adding.
3. Error: `Not Found`

**Expected behavior**
I am expecting I can go to monitored movie to see details, files, search or manage renaming.

**Screenshoots**

![2019-10-01 17_52_03-Add Movie - Radarr](https://user-images.githubusercontent.com/6087244/65978682-7615f300-e474-11e9-90a8-a4fbbcc70aef.png)

![2019-10-01 17_51_34-Not Found - Radarr](https://user-images.githubusercontent.com/6087244/65978755-89c15980-e474-11e9-94ed-160f3ddf36d5.png)

**System version**
*Linux:* `Linux server 4.19.0-6-amd64 #1 SMP Debian 4.19.67-2 (2019-08-28) x86_64 GNU/Linux`
*Mono:*
```
Mono JIT compiler version 6.0.0.334 (tarball Thu Sep 12 21:30:59 UTC 2019)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
        TLS:           __thread
        SIGSEGV:       altstack
        Notifications: epoll
        Architecture:  amd64
        Disabled:      none
        Misc:          softdebug
        Interpreter:   yes
        LLVM:          yes(600)
        Suspend:       hybrid
        GC:            sgen (concurrent by default)
```
**Debug Logs**
```bash
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Req: 129 [GET] /api/config/ui
Oct 01 17:37:20 server mono[17817]: [Trace] ConfigService: Using default config value for 'firstdayofweek' defaultValue:'1'
Oct 01 17:37:20 server mono[17817]: [Trace] ConfigService: Using default config value for 'calendarweekcolumnheader' defaultValue:'ddd M/D'
Oct 01 17:37:20 server mono[17817]: [Trace] ConfigService: Using default config value for 'shortdateformat' defaultValue:'MMM D YYYY'
Oct 01 17:37:20 server mono[17817]: [Trace] ConfigService: Using default config value for 'longdateformat' defaultValue:'dddd, MMMM D YYYY'
Oct 01 17:37:20 server mono[17817]: [Trace] ConfigService: Using default config value for 'timeformat' defaultValue:'h(:mm)a'
Oct 01 17:37:20 server mono[17817]: [Trace] ConfigService: Using default config value for 'showrelativedates' defaultValue:'True'
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Res: 129 [GET] /api/config/ui: 200.OK (6 ms)
Oct 01 17:37:20 server mono[17817]: [Debug] Api: [GET] /api/config/ui: 200.OK (6 ms)
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Req: 130 [GET] /api/system/status
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Res: 130 [GET] /api/system/status: 200.OK (4 ms)
Oct 01 17:37:20 server mono[17817]: [Debug] Api: [GET] /api/system/status: 200.OK (4 ms)
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Req: 131 [GET] /api/queue?sort_by=timeleft&order=asc
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Req: 132 [GET] /api/profile
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Res: 131 [GET] /api/queue?sort_by=timeleft&order=asc: 200.OK (5 ms)
Oct 01 17:37:20 server mono[17817]: [Debug] Api: [GET] /api/queue?sort_by=timeleft&order=asc: 200.OK (5 ms)
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Res: 132 [GET] /api/profile: 200.OK (7 ms)
Oct 01 17:37:20 server mono[17817]: [Debug] Api: [GET] /api/profile: 200.OK (7 ms)
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Req: 133 [GET] /api/health
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Req: 134 [GET] /api/movie?page=1&pageSize=250&sortKey=sortTitle&sortDir=asc&filterKey=null&filterValue=null
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Res: 133 [GET] /api/health: 200.OK (3 ms)
Oct 01 17:37:20 server mono[17817]: [Debug] Api: [GET] /api/health: 200.OK (3 ms)
Oct 01 17:37:20 server mono[17817]: [Trace] Owin: SQLite warning (284): automatic index on AlternativeTitles(MovieId)
Oct 01 17:37:20 server mono[17817]: [Trace] Owin:
Oct 01 17:37:20 server mono[17817]:
Oct 01 17:37:20 server mono[17817]: [Trace] Owin: SQLite warning (284): automatic index on MovieFiles(MovieId)
Oct 01 17:37:20 server mono[17817]: [Trace] Owin:
Oct 01 17:37:20 server mono[17817]:
Oct 01 17:37:20 server mono[17817]: [Trace] Owin: SQLite warning (284): automatic index on AlternativeTitles(MovieId)
Oct 01 17:37:20 server mono[17817]: [Trace] Owin:
Oct 01 17:37:20 server mono[17817]:
Oct 01 17:37:20 server mono[17817]: [Trace] Owin: SQLite warning (284): automatic index on MovieFiles(MovieId)
Oct 01 17:37:20 server mono[17817]: [Trace] Owin:
Oct 01 17:37:20 server mono[17817]:
Oct 01 17:37:20 server mono[17817]: [Trace] Owin: SQLite warning (284): automatic index on AlternativeTitles(MovieId)
Oct 01 17:37:20 server mono[17817]: [Trace] Owin:
Oct 01 17:37:20 server mono[17817]:
Oct 01 17:37:20 server mono[17817]: [Trace] Owin: SQLite warning (284): automatic index on MovieFiles(MovieId)
Oct 01 17:37:20 server mono[17817]: [Trace] Owin:
Oct 01 17:37:20 server mono[17817]:
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Res: 134 [GET] /api/movie?page=1&pageSize=250&sortKey=sortTitle&sortDir=asc&filterKey=null&filterValue=null: 200.OK (120 ms)
Oct 01 17:37:20 server mono[17817]: [Debug] Api: [GET] /api/movie?page=1&pageSize=250&sortKey=sortTitle&sortDir=asc&filterKey=null&filterValue=null: 200.OK (120 ms)
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Req: 135 [GET] /api/movie?page=1&pageSize=-1&sortKey=sortTitle&sortDir=asc&filterKey=filterKey&filterValue=filterValue&filterType=filterType
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Req: 136 [GET] /api/command
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Req: 137 [GET] /api/language
Oct 01 17:37:20 server mono[17817]: [Trace] CommandQueueManager: Getting started commands
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Res: 136 [GET] /api/command: 200.OK (4 ms)
Oct 01 17:37:20 server mono[17817]: [Debug] Api: [GET] /api/command: 200.OK (4 ms)
Oct 01 17:37:20 server mono[17817]: [Trace] Owin: SQLite warning (284): automatic index on AlternativeTitles(MovieId)
Oct 01 17:37:20 server mono[17817]: [Trace] Owin:
Oct 01 17:37:20 server mono[17817]:
Oct 01 17:37:20 server mono[17817]: [Trace] Owin: SQLite warning (284): automatic index on MovieFiles(MovieId)
Oct 01 17:37:20 server mono[17817]: [Trace] Owin:
Oct 01 17:37:20 server mono[17817]:
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Req: 138 [GET] /api/tag
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Res: 137 [GET] /api/language: 200.OK (5 ms)
Oct 01 17:37:20 server mono[17817]: [Debug] Api: [GET] /api/language: 200.OK (5 ms)
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Res: 138 [GET] /api/tag: 200.OK (15 ms)
Oct 01 17:37:20 server mono[17817]: [Debug] Api: [GET] /api/tag: 200.OK (15 ms)
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Req: 139 [GET] /api/movies/discover/lists
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Req: 140 [GET] /api/series
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Res: 140 [GET] /api/series: 200.OK (4 ms)
Oct 01 17:37:20 server mono[17817]: [Debug] Api: [GET] /api/series: 200.OK (4 ms)
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Res: 139 [GET] /api/movies/discover/lists: 200.OK (13 ms)
Oct 01 17:37:20 server mono[17817]: [Debug] Api: [GET] /api/movies/discover/lists: 200.OK (13 ms)
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Req: 141 [GET] /api/movie/titleslug/we-were-soldiers-10590
Oct 01 17:37:20 server mono[17817]: [Trace] Owin: SQLite warning (284): automatic index on MovieFiles(MovieId)
Oct 01 17:37:20 server mono[17817]: [Trace] Owin:
Oct 01 17:37:20 server mono[17817]:
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Res: 141 [GET] /api/movie/titleslug/we-were-soldiers-10590: 200.OK (11 ms)
Oct 01 17:37:20 server mono[17817]: [Debug] Api: [GET] /api/movie/titleslug/we-were-soldiers-10590: 200.OK (11 ms)
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Req: 142 [GET] /api/health
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Res: 142 [GET] /api/health: 200.OK (3 ms)
Oct 01 17:37:20 server mono[17817]: [Debug] Api: [GET] /api/health: 200.OK (3 ms)
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Req: 143 [GET] /api/queue?sort_by=timeleft&order=asc
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Res: 143 [GET] /api/queue?sort_by=timeleft&order=asc: 200.OK (4 ms)
Oct 01 17:37:20 server mono[17817]: [Debug] Api: [GET] /api/queue?sort_by=timeleft&order=asc: 200.OK (4 ms)
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Res: 135 [GET] /api/movie?page=1&pageSize=-1&sortKey=sortTitle&sortDir=asc&filterKey=filterKey&filterValue=filterValue&filterType=filterType: 200.OK (375 ms)
Oct 01 17:37:20 server mono[17817]: [Debug] Api: [GET] /api/movie?page=1&pageSize=-1&sortKey=sortTitle&sortDir=asc&filterKey=filterKey&filterValue=filterValue&filterType=filterType: 200.OK (375 ms)
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Req: 144 [GET] /Content/Images/favicon/favicon.ico?h=5rhS3DvgQdQTAys3GmDWXw
Oct 01 17:37:20 server mono[17817]: [Trace] Http: Res: 144 [GET] /Content/Images/favicon/favicon.ico?h=5rhS3DvgQdQTAys3GmDWXw: 200.OK (3 ms)
Oct 01 17:37:29 server mono[17817]: [Trace] Scheduler: Pending Tasks: 0
```

**Comment by fryfrog** [comment]
Try in an incognito window. Are you doing reverse proxy? Nothing stands out to me, but I've seen this issue on my own setup and for others in support and it has always ended up being something browser related.

**Comment by zetneteork** [comment]
I've tried:
* clear cache
* clear cookies
* different browser
* access radarr dicectly without proxy
* Housekeeping

Many movies are ok. But some of them are unable to locate via search. 
It is only possible to access them via `Add Movies` search them and use button `Already Exist`.
Than it navigates under the URL `http://radarr_server_url:7878/radarr/movies/we-were-soldiers-10590` and **404** appears.

**Comment by zetneteork** [comment]
Tested right now on new host. Fresh new installation. Empty DB.
Manual mass import. And result is a same. 404

**Comment by zetneteork** [comment]
This issue is occurring more and more frequently.

**Comment by zetneteork** [comment]
This issue had not been resolved.
But is still pressing. 
Suggesting to leave it open

**Comment by dm3ch** [comment]
This bug also exists in 2nd major version. :(

**Comment by Qstick** [comment]
Should be fixed in 1f72e1de7439951bebba41b3a1f1fcc0ea311a32, version 0.2.0.1480
